### DISCUSSION TITLE
Ruleset Idea

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21695] [Ideas]
Anyone who interested in the huge develompent: there is a game called "Rocksmith" where you can play the songs by playing on a real guitar. I propose to create smth similar within osu! game as a ruleset with a guitar connection to a PC and own editor.

**Comment by Axilotl17** [comment]
Users can create rulesets on their own, as [touhosu](https://github.com/EVAST9919/touhosu). and if i recall correctly, osu!lazer already supports MIDI connections, so it wouldn't be a huge stretch to plug in a MIDI guitar. however, your average electric guitar would be extremely difficult to integrate into the game. 

Editors for custom rulesets will be here eventually. (if they aren't already here.)
